models/us_harvard_cbdb/code/gen_dbt.py

```
@@ -0,0 +1,238 @@
+        alias="{name}",
+        materialized="table",
+    )
+}}}}
+
+
+select
+{body}
+from
+    {{{{ set_datalake_project("{DS}_staging.{name}") }}}}
+    as t
+'''
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)